lib/csv2avro.rb

```diff
@@ -14,7 +24,11 @@ def initialize(options)
   end
 
   def convert
-    Converter.new(reader, writer, bad_rows_writer, error_writer, options, schema: schema).convert
+    log_writer.info(event: Log::Event.new('started_processing', true, {filename: File.basename(input_path)}))
+
+    Converter.new(reader, writer, bad_rows_writer, log_writer, input_path, options, schema: schema).convert
+
+    log_writer.info(event: Log::Event.new('processing_done', true, {filename: File.basename(input_path)}))
```

Comment:
We never discussed naming of the event, but all event names should be consistent. If we have a `started_processing` event, we should have a `finished_processing` event. Looking at the code, we are actually `convert`ing here, so `started_converting` and `finished_converting` makes the most sense to me.

We can also include metrics on these events, such as number of rows processed, number of invalid rows, parse errors, etc on `finished_processing`. It would be very easy to work with those in downstream systems.